src/main/java/org/opentripplanner/model/FeedScopedId.java

```diff
@@ -105,4 +108,14 @@ public static String convertToString(FeedScopedId aid) {
     public static String concatenateId(String feedId, String id) {
         return feedId + ID_SEPARATOR + id;
     }
+
+    /**
+     * Converts a string consisting of concatenated FeedScopedIds to a Set
+     */
+    public static HashSet<FeedScopedId> convertFromConcatenatedString(String s) {
```

Comment:
I would call this function `convertFromListOfIds` or `parseListOfIds. I prefer `parse` over `convert`, but we already used _convert_ for the single element version, so it should be consistent.

Reply:
Changed both of these to start with convert